monitoring/harmony-monitor/blockchain-watchdog/html-generators.go

```diff
@@ -90,6 +90,22 @@ hr:after {
         </div>
       </div>
       <hr/>
+      <div class="build-stat">
+        {{ with (index .Summary "chain-config") }}
+        {{range $key, $value := .}}
+          <div class="flex-col center stat-box">
+            <a href="#shard-{{$key}}">Shard-{{$key}}</a>
```

Comment:
for each of these html tags in the `stat-box`, wrap each the key and value into their own <span>, then on the <p> that holds the two <span> tags, attach to it to have css of `display:flex`, and `justify-content:space-between;`, example: 

<p class="row"><span>Chain ID: </span> <span>  {{index $value "chain-id"}} </span> </p>